MIP/mip-58/README.md

```diff
@@ -0,0 +1,147 @@
+# MIP-28: Simplified Bridge Design
+
+- **Description**: Proposes a simplified bridge design to address inefficiencies, security risks, and high costs in the current implementation.
+- **Authors**: [Author](mailto:primata@movementlabs.xyz)
+- **Desiderata**: [MD-21](../MD/md-21)
+
+## Abstract
+
+This proposal advocates for replacing the current HTLC-based bridge design, which requires four transactions and introduces significant complexity, with a simpler and more efficient two-transaction mechanism. The proposed design reduces costs, enhances security, minimizes user friction, and avoids potential exploits caused by refund logic. It addresses long-standing issues such as reliance on sponsored transactions, outdated audits, and unimplemented fee mechanisms speeding the process to achieve a fully functional bridge that movement yet has been unable to complete, and lots of under issues raised and still pending.
+
+## Motivation
+
+The current bridge design poses numerous challenges, including inefficiency, cost, and user frustration:
+1. **Transaction Complexity**: Requires four interactions:
+   - Two by the user (initiation and finalization).
+   - One by the relayer to complete the transaction.
+   - One by a third party (tasked by the relayer) to finalize refunds in case of failure.
+   - If any of these fails, users and the protocol face losses.
+2. **High Cost**: 
+   - The multi-transaction setup is expensive for both users and the protocol. 
+   - Misestimations in fee calculations could result in significant losses.
+   - Example: At 16 gwei, bridging costs $20 for 10k transactions. If fees are miscalculated by 25%, this leads to a $25k loss. At 100 gwei, the loss escalates dramatically.
+3. **Unfriendly User Experience**:
+   - The current design requires users to have funds on the destination chain for finalization. This increases friction and discourages adoption.
+   - Sponsored transactions are essential but remain unimplemented, leaving users stranded without funds.
+4. **Security Risks**:
+   - Relayer keys are a critical point of failure.
+   - Refund logic introduces vulnerabilities where attackers could exploit the rate limit and relayer downtime.
+   - For instance, if the relayer fails to finalize on the initiator, a malicious actor could:
+     - Take over the refund keys.
+     - Exploit both directions of the bridge (e.g., ETH→MOVE and MOVE→ETH) repeatedly, draining funds.
+5. **Maintenance Burden**:
+   - The current audit is outdated and does not reflect the significant changes made since.
+   - The current design requires a complete UI/UX overhaul, adding complexity and delay.
+6. **Unnecessary Complexity**:
+   - HTLC-based bridges are largely abandoned in favor of simpler, more effective designs.
+   - Examples like the Consensys HTLC bridge demonstrate the pitfalls of such approaches.
```

Comment:
Here's a link to an article by Consensys citing Connext’s Amarok upgrade as an example of moving away from a so-called atomic swap (HTLC) design: https://entethalliance.org/crosschain-bridges-overview-where-we-are-now/

You could also link to the old Consensys HTLC bridge code https://github.com/Consensys/htlc-bridge to show that it is no longer in development in favor of another model.